CleanCodeJN.GenericApis.Sample/Program.cs

```diff
@@ -16,6 +16,7 @@
 });
 
 builder.Services.RegisterRepositoriesCommandsWithAutomaticMapping<MyDbContext>();
+builder.Services.AddApiImplementations();
```

Comment:
This should be inside of the Register Extension, if it is needed. I think you don´t need it, because of my other comment.